user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQILogger.md

```diff
@@ -0,0 +1,79 @@
+---
+uid: GQI_IGQILogger
+---
+
+# IGQILogger interface
+
+## Definition
+
+- Namespace: `Skyline.DataMiner.Analytics.GenericInterface`
+- Assembly: `SLAnalyticsTypes.dll`
+
+Contains functionality to log messages and exceptions. The log files will be placed within the `C:\Skyline DataMiner\Logging\GQI\Extensions` folder.
+
+## Properties
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| MinimumLogLevel | GQILogLevel | Set or get the current minimum log level. The default value is `Information`. |
+
+GQILogLevel exposes the following log levels:
+- **Verbose**: Detailed information for debugging, usually not used in production.
+- **Debug**: Internal system events for debugging and troubleshooting.
+- **Information**: General operational messages to track application behavior.
+- **Warning** Indicates potential issues that may need attention.
+- **Error**: Signals unexpected failures or malfunctions in the application.
+- **Fatal**: Critical errors requiring immediate action, often leading to application termination.
+
+## Methods
+
+> [!NOTE]
+> The IGQILogger logger methods are inspired by [Serilog](https://github.com/serilog/serilog).
```

Comment:
```suggestion
> The IGQILogger methods are inspired by [Serilog](https://github.com/serilog/serilog).
```

I'm wondering if we also need to mention that they actually use Serilog in the background and so the message will be interpreted as a Serilog template.

Reply:
Not the case anymore